.github/workflows/update.yml

```diff
@@ -0,0 +1,78 @@
+name: Update
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout l10n:main
+        uses: actions/checkout@v4
+        with: { ref: main }
+      - name: Refresh l10n:update
+        run: git push origin HEAD:update || true
+  branches:
+    needs: prepare
+    strategy:
+      max-parallel: 1
+      matrix:
+        ref: [master, beta, release, esr115]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout l10n:update
+        uses: actions/checkout@v4
+        with: { ref: update, path: l10n }
+      - name: Checkout firefox:${{ matrix.ref }}
+        uses: actions/checkout@v4
+        with:
+          repository: mozilla/gecko-dev
+          ref: ${{ matrix.ref }}
+          path: firefox
+          sparse-checkout: |
+            browser/branding/official/locales
+            browser/extensions/formautofill/locales
+            browser/extensions/report-site-issue/locales
+            browser/locales
+            devtools/client/locales
+            devtools/shared/locales
+            devtools/startup/locales
+            dom/locales
+            mobile/android/branding/official/locales
+            mobile/android/locales
+            netwerk/locales
+            security/manager/locales
+            toolkit/locales
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          cache: pip
+          cache-dependency-path: l10n/_scripts/requirements.txt
+      - run: pip install -r l10n/_scripts/requirements.txt
+      - run: python -m _scripts.update ${{ matrix.ref }}
```

Comment:
Out of curiosity: why the choice of invoking this as a module instead of running it as a script? 

Besides that, I personally have a preference for name parameters (no risk of confusion).

Reply:
Because running as a script would change how local imports would be resolved. Effectively, I'm looking to avoid a footgun in later refactorings.

If we do need more than one parameter, then I agree that they should be named.